### PR TITLE
Change Position of Delete Column in ReviewEntries

### LINKED ISSUE #375: Change the position of delete icon next to each glosses in ReviewEntries.

### PR DESCRIPTION
**Pull request by shyung18**
Position the delete button next to each glosses instead of after the recorder.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/thecombine/343)

Closes #375

### CODE REVIEW

File: src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx [hunk 1]
```diff
@@ -122,6 +122,30 @@ const columns: Column[] = [
       return compare;
     },
   },
+  {
```
Comment (johnthagen) [inline src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx:125]: @shyung18 Our CI validates that Prettier has been run on all code before it's accepted into `master`.

```
[09:31:37]
Step 4/9: Frontend Format (npm run prettier-check) (Node.js NPM) (6s)
[09:31:37]
[Step 4/9] npm run-script prettier-check (6s)
[09:31:37]
[npm run-script prettier-check] Executing npm via wrapping shell script
[09:31:37]
[npm run-script prettier-check] Starting: /var/lib/TeamCity/agent/temp/agentTmp/wrapper860718286964877718.sh run-script prettier-check
[09:31:37]
[npm run-script prettier-check] in directory: /var/lib/TeamCity/agent/work/47a4bd68a4fe076f
[09:31:37]
[npm run-script prettier-check] 
[09:31:37]
[npm run-script prettier-check] > thecombine@0.1.0 prettier-check /var/lib/TeamCity/agent/work/47a4bd68a4fe076f
[09:31:37]
[npm run-script prettier-check] > prettier --check "./src/**/*.{js,jsx,ts,tsx,json}"
[09:31:37]
[npm run-script prettier-check] 
[09:31:37]
[npm run-script prettier-check] Checking formatting...
[09:31:40]
[npm run-script prettier-check] src/components/ProjectSettings/ProjectSettingsComponent.tsx
[09:31:41]
[npm run-script prettier-check] src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx
[09:31:43]
[npm run-script prettier-check] Code style issues found in the above file(s). Forgot to run Prettier?
```

Please run `npm run prettier` before committing/pushing to address this.

Comment (johnthagen) [reply src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx:125]: Please see #347 for instructions on how to set up VS Code to automatically format code with Prettier on save.

Comment (shyung18) [reply src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx:125]: Got it! Thanks!

Comment (shyung18) [reply src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx:125]: I ran Prettier, but the tests are still failing. Any idea why?

Comment (johnthagen) [reply src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/CellColumns.tsx:125]: Looks like the frontend unit tests are failing. Please try running `npm run test` to see output on your local machine. From the CI:

```
...
[03:39:12][npm run-script test:ci] tests/CellColumns.test.tsx
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Renders vernacular without crashing
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Returns true when searching a word for an extant gloss
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Returns false when searching a word for an extant gloss
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Properly sorts a list by senses
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Returns true when searching a word for an extant domain
[03:39:12][Tests cell column functions Returns true when searching a word for an extant domain] FAILED
[03:39:12][Tests cell column functions Returns true when searching a word for an extant domain] Error: Failed: "There is no customFilterAndSearch on column[2]: domains.
Did the domains get moved to a different column?"
    at Env.fail (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmine/Env.js:778:61)
    at fail (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmine/jasmineLight.js:149:23)
    at Object.it (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/tests/CellColumns.test.tsx:126:7)
    at Object.asyncJestTest (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Returns true when searching a word for an extant domain
[03:39:12][Tests cell column functions Returns true when searching a word for an extant domain] FAILED
[03:39:12][Tests cell column functions Returns true when searching a word for an extant domain] Error: Failed: "There is no customFilterAndSearch on column[2]: domains.
Did the domains get moved to a different column?"
    at Env.fail (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmine/Env.js:778:61)
    at fail (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmine/jasmineLight.js:149:23)
    at Object.it (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/tests/CellColumns.test.tsx:135:7)
    at Object.asyncJestTest (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
[03:39:12][tests/CellColumns.test.tsx] Tests cell column functions Properly sorts a list by domains
[03:39:12][Tests cell column functions Properly sorts a list by domains] FAILED
[03:39:12][Tests cell column functions Properly sorts a list by domains] Error: Failed: "There is no customSort on column[2]: domains.
Did the domains get moved to a different column?"
    at Env.fail (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmine/Env.js:778:61)
    at fail (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmine/jasmineLight.js:149:23)
    at Object.it (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/src/goals/ReviewEntries/ReviewEntriesComponent/CellComponents/tests/CellColumns.test.tsx:149:7)
    at Object.asyncJestTest (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/TeamCity/agent/work/47a4bd68a4fe076f/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
```

**Comment by johnthagen** [comment]
@shyung18 I merged master into your branch to address a new CI build step we are using for the backend (should not affect your code, just wanted to give you a heads up to pull the latest from your branch).

**Pull request merged**